src/components/Mapper/coordinates.js

```diff
@@ -0,0 +1,1316 @@
+export const coordinates = [
```

Comment:
Consider putting coordinates into a separate JSON file. This way data is kept separate from business logic

Reply:
Good idea. Moving that over.